components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/StandaloneJobOrchestrator.cs

```diff
@@ -491,9 +491,9 @@ IJsonSerializer jsonSerializer
             job.WriterGroup.DataSetWriters.ForEach(d => {
                 d.DataSet.ExtensionFields ??= new Dictionary<string, string>();
                 d.DataSet.ExtensionFields["PublisherId"] = jobId;
-                d.DataSet.ExtensionFields["DataSetWriterId"] = d.DataSetWriterId;
+                d.DataSet.ExtensionFields["DataSetWriterId"] = d.DataSetWriterName;
```

Comment:
would it make sense to rename DataSetWriterId to DataSetWriterName overall? to avoid confusions?

Reply:
I think we should eventually. I want to make sure we do not break backcompat though.  Let me take an action item to follow up.